internal/handler/response.go

```diff
@@ -2,16 +2,23 @@ package handler
 
 import (
 	"encoding/json"
+	"fmt"
 	"net/http"
 )
 
 func writeResponse(w http.ResponseWriter, code int, v interface{}) {
-	b, err := json.Marshal(v)
-	if err != nil {
+	switch t := v.(type) {
+	case error:
 		w.WriteHeader(http.StatusInternalServerError)
-		w.Write([]byte(`{"error":"Internal server error"}`))
-		return
+		w.Write([]byte(fmt.Sprintf("%v", t)))
```

Comment:
Please see my comment in another [PR](https://github.com/andreipimenov/golang-training-2021/pull/73#discussion_r681552114). The same as here.